core/src/test/java/org/apache/iceberg/TestFastAppend.java

```diff
@@ -114,12 +114,14 @@ public void testEmptyTableAppendFilesAndManifest() throws IOException {
 
     validateManifest(
         snap.allManifests(FILE_IO).get(0),
-        seqs(1, 1),
+        seqs(1L, 1L), // data sequence numbers
```

Comment:
Minor: I'd create two methods, `dataSeqs` and `fileSeqs` for this. That way the code is self-documenting. We do basically the same thing for `ids` and `seqs` currently.

Reply:
I had that originally. I'll switch back.

Reply:
Done.